recipes/sciphin/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# conda on macOS will fall back to libstdc++
```

Comment:
are you sure about that? Conda on OSX will use LLVM.

Reply:
Thanks for pointing this out. I used an old recipe as a template and did not remove these lines. Sorry for that!